shumai/tensor/stats.ts

```diff
@@ -22,12 +22,86 @@ export function collectStats(tensors: Tensor[]) {
     }
     for (const key of Object.keys(t.stats)) {
       if (key in stats) {
+        stats[key].count += t.stats[key].count
         stats[key].time += t.stats[key].time
         stats[key].bytes += t.stats[key].bytes
+        stats[key].ops += t.stats[key].ops
       } else {
         stats[key] = t.stats[key]
       }
     }
   }
   return stats
 }
+
+export type StatsEntry = {
+  count: number
+  time: number
+  bytes: bigint
+  ops: bigint
```

Comment:
let's rename this `flops` everywhere.  Admittedly they're not all floating point, but the nomenclature is common in the literature and it'll prevent confusion with `op` as in "operation"